src/geo/ui/infowindow.js

```diff
@@ -164,36 +160,28 @@ var Infowindow = View.extend({
     }
   },
 
-  _renderLoader: function () {
+  _renderCoverLoader: function () {
+    var $loader = $('<div class="CDB-Loader js-loader"></div>');
```

Comment:
_Totally optional: You could create the element as $('<div>').addClass('CDB-Loader js-loader')_